zmon-security-common/src/main/java/org/zalando/zmon/security/authority/ZMONRoleToAuthority.java

```diff
@@ -0,0 +1,31 @@
+package org.zalando.zmon.security.authority;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.GrantedAuthority;
+
+import com.google.common.collect.ImmutableSet;
+
+public final class ZMONRoleToAuthority {
+    private static final Logger log = LoggerFactory.getLogger(ZMONRoleToAuthority.class);
+    private static final Map<String, GrantedAuthorityFactory> factories = new HashMap<>();
+
+    static {
+        factories.put(ZMonRole.ADMIN.getRoleName(), ZMonAdminAuthority.FACTORY);
```

Comment:
Does it make sense to have the FACTORY mapping within the ZMonRole array instead of in a disconnected class?